synchronization/src/main/java/de/cyface/synchronization/HttpConnection.java

```diff
@@ -69,6 +67,12 @@
      */
     final static String TAIL = "\r\n--" + BOUNDARY + "--\r\n";
 
+    /**
+     * The status code returned when the MultiPart request is erroneous, e.g. when there is not exactly onf file or a
+     * syntax error.
+     */
+    final static int HTTP_ENTITY_NOT_PROCESSABLE = 422;
```

Comment:
Den Code gibt's nicht in HttpURLConnection wo wir die anderen statisch verlinken